grouper/fe/forms.py

```diff
@@ -333,13 +333,15 @@ class UserPasswordForm(Form):
 
 
 class ServiceAccountCreateForm(Form):
+    # Allow either NAME or SERVICE_ACCOUNT validation since we allow people to specify just the
```

Comment:
That was a great catch. It turns out that it is AND, but we were never checking the results of validation because there was a missing `return`. Fix coming as another diff on top of this one.

(No idea why we aren't using `Regex`; maybe it didn't exist when this was first written. We should switch.)